crates/iceberg/src/spec/manifest.rs

```diff
@@ -1052,6 +1052,89 @@ pub struct DataFile {
     pub(crate) sort_order_id: Option<i32>,
 }
 
+impl DataFile {
+    /// Get the content type of the data file (data, equality deletes, or position deletes)
+    pub fn content(&self) -> DataContentType {
```

Comment:
We can have `#[inline]` attribute to give compiler a hint, but this is optional.

Reply:
Thanks @liurenjie1024 , You mean to `#[inline]` all pub methods on DataFile? 
I actually thought about this, but then saw that inline was not used on other pub struct methods (such as Table, Namespace, etc) so preferred to maintain consistency. 
If you think its better then I will do this

Reply:
I think it's fine for now, and we could add it later when necessary, thanks!